ocs_ci/deployment/tnf.py

```
@@ -0,0 +1,308 @@
+        """
+        Pre-requisites for TNF deployment
+        """
+        super().deploy_prereq()
+        logger.info("Verifying TNF deployment prerequisites...")
+
+        # Verify two-node cluster topology
+        if not verify_tnf_cluster_topology():
+            raise UnexpectedDeploymentConfiguration(
+                "Cluster does not have DualReplica topology. "
+                "TNF deployment requires a two-node cluster."
+            )
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)